cmake/extern/ExternalSTRUMPACK.cmake

```diff
@@ -22,6 +67,13 @@ if(PALACE_STRUMPACK_WITH_BUTTERFLYPACK)
     "-Denable_openmp=${PALACE_WITH_OPENMP}"
     "-DTPL_SCALAPACK_LIBRARIES=${SCALAPACK_LIBRARIES}"
   )
+  # if(PALACE_STRUMPACK_WITH_ZFP)
+  #   string(REPLACE ";" "$<SEMICOLON>" _ZFP_LIBRARIES "${_ZFP_LIBRARIES}")
+  #   list(APPEND BUTTERFLYPACK_OPTIONS
+  #     "-DTPL_ZFP_LIBRARIES=${_ZFP_LIBRARIES}"
+  #     "-DTPL_ZFP_INCLUDE=${CMAKE_INSTALL_PREFIX}/include"
+  #   )
+  # endif()
```

Comment:
Debris?

Reply:
I would prefer to leave this in, in case eventually people are interested in using ZFP with ButterflyPACK this is how it would be done. Right now this was just added experimentally so I'm not inclined to support it in `main` yet.